rekor-v2-spec.md

```diff
@@ -0,0 +1,241 @@
+signing event using the data contained in the record. In general, it will
+contain some representation of the data signed, the signature(s), and the key
+material used to verify the signature(s).
+
+Validation metadata should include verification material which can verify the
+signature(s). Validation metadata will commonly be a code-signing certificate or
+a public key.
+
+#### 4.1.4 Canonicalization
+
+The transparency service will perform type-specific canonicalization on records
+to de-duplicate equivalent entries. An equivalent entry is one that represents
+the same signing event. Sources of duplication include encoding schemes of the
+public key or signature. For example, the transparency service must de-duplicate
+an entry that encodes the public key with DER-encoding from one that uses the
+PEM-encoding. Canonicalization must not remove any necessary information to
+validate the signature. The service must store enough information in the Merkle
+tree leaf for an interested party to perform the same record validation to
+ensure entry auditability.
+
+### 4.2 HTTP and gRPC API
+
+The Rekor v2 write API is accessible over HTTP or gRPC, and the read-only tiles
+API and checkpoint are accessible over HTTP. Refer to the [protobuf
+specification](https://github.com/sigstore/rekor-tiles/tree/main/api/proto) for
+the exact schema, and the [clients
+guide](https://github.com/sigstore/rekor-tiles/blob/main/CLIENTS.md) for
+examples.
+
+### 4.3 Checkpoint Format
+
+The Merkle tree checkpoint, accessible via a GET request to the
+`/api/v2/checkpoint` endpoint, follows the [C2SP Transparency Log
+Checkpoints](https://github.com/C2SP/C2SP/blob/main/tlog-checkpoint.md)
+specification. The checkpoint MAY additionally include witness signatures as
+specified in the [Transparency Log Witness
+Protocol](https://github.com/C2SP/C2SP/blob/main/tlog-witness.md).
+
+## 5. Client Workflows
+
+### 5.1 Signers
+
+A client signing an artifact uploads the entry record, formatted as either a
+[HashedRekordRequestV002](https://github.com/sigstore/rekor-tiles/blob/f3cba09c2f92f1d2d5a7ca2b5694b66941a82d41/api/proto/rekor/v2/hashedrekord.proto#L31C9-L31C32)
+or
+[DSSERequestV002](https://github.com/sigstore/rekor-tiles/blob/f3cba09c2f92f1d2d5a7ca2b5694b66941a82d41/api/proto/rekor/v2/dsse.proto#L32C9-L32C24) to the
+`/api/v2/log/entries` (HTTP) or `dev.sigstore.rekor.v2.Rekor.CreateEntry` (gRPC)
+service. The client receives, as a response, a
+[TransparencyLogEntry](https://github.com/sigstore/protobuf-specs/blob/4df5baadcdb582a70c2bc032e042c0a218eb3841/protos/sigstore_rekor.proto#L94),
+which contains the latest checkpoint and an inclusion proof for the entry.
+Because entries are batched before being integrated, and because checkpoints are
+published only at regular intervals, the client may have to wait for a few
+seconds before receiving a response, depending on the configuration of the
+service.
+
+### 5.2 Artifact Verifiers
```

Comment:
Unfortunately "verifiers" is overloaded, both with signature verification and entry verification as defined by the claimant model. Can we use both the claimant model term ("believer") and say that means artifact signature verification?

Reply:
> say that means artifact signature verification?

This section is really more about verifying log inclusion than verifying the signature